examples/rust-generate-crate/__snapshots__/index.spec.ts.snap

```diff
@@ -52,12 +52,12 @@ Array [
   "// Members represents a union of types: String, f64, bool
 #[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
 pub enum Members {
-    #[serde(rename=\\"Members0\\")]
-    Members0(String),
-    #[serde(rename=\\"Members1\\")]
-    Members1(f64),
-    #[serde(rename=\\"Members2\\")]
-    Members2(bool),
+    #[serde(rename=\\"MembersOneOf_00\\")]
```

Comment:
We might have to take a double check down the line after this PR, https://github.com/asyncapi/modelina/pull/869, and https://github.com/asyncapi/modelina/pull/876 how we can properly name schemas for models.

cc @leigh-johnson